llvm_util/llvm2alive.cpp

```diff
@@ -332,9 +332,15 @@ class llvm2alive_ : public llvm::InstVisitor<llvm2alive_, unique_ptr<Instr>> {
       if (!op)
         return error(i);
 
-      // TODO: support for struct indexing
-      if (I.getStructTypeOrNull())
-        return error(i);
+      if (auto structTy = I.getStructTypeOrNull()) {
+	auto const0 = llvm::ConstantInt::get(llvm::Type::getInt32Ty(i.getType()->getContext()), 0);
+	auto const1 = llvm::ConstantInt::get(llvm::Type::getInt32Ty(i.getType()->getContext()), 1);
```

Comment:
Can factor out `llvm::Type::getInt32Ty(i.getType()->getContext())` as both lines have the expression.
You'd like to check the indentation as well.

Reply:
done.